pkg/cmd/grafana-cli/commands/secretsmigrations/reencrypt_secrets.go

```diff
@@ -256,9 +261,10 @@ func ReEncryptSecrets(_ utils.CommandLine, runner runner.Runner) error {
 		reencrypt(*manager.SecretsService, *xorm.Session)
 	}{
 		simpleSecret{tableName: "dashboard_snapshot", columnName: "dashboard_encrypted"},
-		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_access_token"}},
-		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_refresh_token"}},
-		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_token_type"}},
+		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_access_token"}, false},
+		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_refresh_token"}, false},
+		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_token_type"}, false},
+		b64Secret{simpleSecret{tableName: "secrets", columnName: "value"}, true},
```

Comment:
```suggestion
		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_access_token"}},
		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_refresh_token"}},
		b64Secret{simpleSecret{tableName: "user_auth", columnName: "o_auth_token_type"}},
		b64Secret{simpleSecret{tableName: "secrets", columnName: "value"}, hasUpdatedColumn: true},
```

Reply:
Mmm.. I intentionally suggested to be verbose (to explicitly say the meaning of the boolean), but only when actually worthful (only on positive values) -due to the verbosity-, but I didn't know it cause issues (I still cannot see what's wrong with this approach, honestly).

Then, just ignore my suggestion and merge it as-is! :)

Reply:
I guess it's not that verbose, verbose is clearer so it's better.

Done on d1e45b8

Reply:
Great, thanks @guicaulada! 👌🏻